README.md

```diff
@@ -3,47 +3,44 @@
   <h1 align="center">Critters</h1>
 </p>
 
-> Critters is a Webpack plugin that inlines your app's [critical CSS] and lazy-loads the rest.
+> Critters is a plugin that inlines your app's [critical CSS] and lazy-loads the rest.
 
-## critters-webpack-plugin [![npm](https://img.shields.io/npm/v/critters-webpack-plugin.svg?style=flat)](https://www.npmjs.org/package/critters-webpack-plugin)
+## critters [![npm](https://img.shields.io/npm/v/critters-webpack-plugin.svg?style=flat)](https://www.npmjs.org/package/critters)
 
-It's a little different from [other options](#similar-libraries), because it **doesn't use a headless browser** to render content.  This tradeoff allows Critters to be very **fast and lightweight**. It also means Critters inlines all CSS rules used by your document, rather than only those needed for above-the-fold content. For alternatives, see [Similar Libraries](#similar-libraries).
+It's a little different from [other options](#similar-libraries), because it **doesn't use a headless browser** to render content. This tradeoff allows Critters to be very **fast and lightweight**. It also means Critters inlines all CSS rules used by your document, rather than only those needed for above-the-fold content. For alternatives, see [Similar Libraries](#similar-libraries).
 
 Critters' design makes it a good fit when inlining critical CSS for prerendered/SSR'd Single Page Applications. It was developed to be an excellent compliment to [prerender-loader](https://github.com/GoogleChromeLabs/prerender-loader), combining to dramatically improve first paint time for most Single Page Applications.
 
 ## Features
 
--   Fast - no browser, few dependencies
--   Integrates with [html-webpack-plugin]
--   Works with `webpack-dev-server` / `webpack serve`
--   Supports preloading and/or inlining critical fonts
--   Prunes unused CSS keyframes and media queries
--   Removes inlined CSS rules from lazy-loaded stylesheets
+- Fast - no browser, few dependencies
+- Integrates with Webpack [critters-webpack-plugin]
+- Supports preloading and/or inlining critical fonts
+- Prunes unused CSS keyframes and media queries
+- Removes inlined CSS rules from lazy-loaded stylesheets
 
 ## Installation
 
 First, install Critters as a development dependency:
 
 ```sh
-npm i -D critters-webpack-plugin
+npm i -D critters
 ```
 
 Then, import Critters into your Webpack configuration and add it to your list of plugins:
 
 ```diff
 // webpack.config.js
-+const Critters = require('critters-webpack-plugin');
++    const Critters = require('critters');
```

Comment:
Might be worth adding a `## Usage With Webpack` section with the old code snippet below this?